imf/distributions.py

```diff
@@ -246,6 +247,120 @@ def ppf(self, x0):
         return ret.reshape(x.shape)
 
 
+class KoenConvolvedPowerLaw(Distribution):
+    """Error-convolved power law.
+
+    A power law over the mass range (m1,m2) with slope -(gamma-1) convolved with
+    a normal distribution of width sigma, as described in Koen & Kondlo 2009.
+    This implementation calculates the PDF and CDF of the distribution at npts
+    evenly log-spaced points and interpolates between the results.
+
+    Arguments:
```

Comment:
This syntax is used in numpy-based documentation to make section headers; you can't use English synonyms if you want the documents to render correctly

```suggestion
    Parameters
    ----------
```